File: Results/DER/Translation/translation_output/w_test/starcoder/codenet/java/python/p00330.py
<fim_prefix>Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
4

Expected output:
128


```

import java.util.Scanner;

public class p00330 {

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        
        int W = scan.nextInt();
        
       
        
        
        System.out.println(W*32);
        
    }
}


```

python code:<fim_suffix><fim_middle>

```python

import sys

W = int(sys.stdin.readline())

print(W*32)

```
<|endoftext|>

File: Results/DER/Translation/translation_output/w_test/starcoder/codenet/python/java/p00330.java
<fim_prefix>Translate the following python code to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
4

Expected output:
128


```
num = int(input())

print(num * 32)

```

java code:<fim_suffix><fim_middle>
```java
import java.util.*;
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num = sc.nextInt();
        System.out.println(num * 32);
    }
}
```

'''
<|endoftext|>
